string arm_name
string object_name
string place_name
geometry_msgs/Pose[] gripper_pose


